### PR TITLE
Fix/#14 eslint rules

### PR DESCRIPTION
**Pull request by liubich**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "extends": ["eslint:recommended", "plugin:react/recommended"],
+    "parser": "babel-eslint",
+    "env": {
+        "browser": true,
+        "node": true,
+        "jest": true
```
Comment (probil) [inline .eslintrc]: you don't have `jest` so remove this line

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "extends": ["eslint:recommended", "plugin:react/recommended"],
+    "parser": "babel-eslint",
+    "env": {
+        "browser": true,
+        "node": true,
```
Comment (probil) [inline .eslintrc]: This app is for browser not for `node`. remove this line

File: .eslintrc [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "extends": ["eslint:recommended", "plugin:react/recommended"],
```
Comment (probil) [inline .eslintrc]: you should probably remove `eslintConfig` from `package.json` then

File: .eslintrc (hunk 1 above)
Comment (liubich) [reply .eslintrc]: @probil, I have a template file for jest and next code inside:
```javascript
it('renders without crashing', () => {
  const div = document.createElement('div');
...
```
Should I remove this file too?

File: .eslintrc (hunk 2 above)
Comment (liubich) [reply .eslintrc]: @liubich, I have next code into serviceWorker.js:
```javascript
const isLocalhost = Boolean(
  window.location.hostname === 'localhost' ||
...
```
Isn't it for node? I'm not sure how I can delete it.

File: .eslintrc (hunk 3 above)
Comment (liubich) [reply .eslintrc]: Ok

File: .eslintrc (hunk 2 above)
Comment (probil) [reply .eslintrc]: It's for service worker
https://developer.mozilla.org/en/docs/Web/API/Service_Worker_API
And it's still browser :)

Comment (probil) [reply .eslintrc]: Just remove line:
```
 "node": true,
```

Comment (liubich) [reply .eslintrc]: @probil, error is in `serviceWorker.js`:
`25:7   error  'process' is not defined  no-undef`

Comment (probil) [reply .eslintrc]: @liubich Add this comment on the top of the file to fix an error:
```
/* global process */
```

**Pull request merged**